LiskHQ/lisk-hub
# Fix not displaying voters and voting - Closes #793

### What was the problem?
Not displaying voters and voting on delegate statistics
### How did I fix it?
`action.data.delegate` changed to `action.delegate`
### How to test it?
Use `segment major cement pepper wedding lunar ready this afford maximum youth amount`
to login into testnet, click on voting delegate transaction and check voters and votes 
### Review checklist
- The PR solves #793 
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)
